docker/ssl/authorization.js

```diff
@@ -138,18 +138,6 @@ function setKey(r) {
         return '';
     }
 
-    if (r.uri.startsWith('/ws') && (!auth || !auth.length)) {
```

Comment:
Does this needs to be updated in the main repo as well?

Reply:
this is update from the main repo